test/Benchmarks/src/Runtime/Managed_Resources.enso

```diff
@@ -0,0 +1,51 @@
+from Standard.Base import all
+import Standard.Base.Runtime.Managed_Resource.Managed_Resource
+import Standard.Base.Runtime.Ref.Ref
+
+from Standard.Test import Bench
+
+from project.Config import extended_tests
+
+options = Bench.options . set_warmup (Bench.phase_conf 2 3) . set_measure (Bench.phase_conf 2 3)
+
+type My_Value
+    Value x:Integer
+
+    foo self =
+        Nothing
+
+type Data
+    Value
+        ~my_values_10000
+
+    create =
+        Data.Value
+            <| make_values 10000
+
+nop_cleanup _ = Nothing
+
+make_values count = 0.up_to count . map (i-> My_Value.Value i)
+
+make_mrs count = make_values count . map (v-> Managed_Resource.register v nop_cleanup)
+
+collect_benches = Bench.build builder->
+    builder.group "Managed_Resource" options group_builder->
+        data = Data.create
```

Comment:
The proper way to deal with _lazy setup_ for `group` is to define own `type` like `Data` with lazy fields and use them when needed. Let's built upon that and:
- define another type `DataForBench` with lazy field
- make `some_mrs` a lazy field in `DataForBench`
- no need for `setup` then - remove it
- in `teardown` `clear_some_mrs`

Reply:
- there already seems to be `teardown` in #9035
- inside of `Group`: https://github.com/enso-org/enso/blob/aa2e77e06f6bec2be003c3437340a8725d736589/distribution/lib/Standard/Test/0.0.0-dev/src/Group.enso#L43
- btw., @Akirathan, how can `Group` be `private` (not listed in the `docs/api` at all) and its `teardown` still be available (as #9035 suggests)?

Reply:
> btw., how can Group be private (not listed in the docs/api at all) and its teardown still be available (as https://github.com/enso-org/enso/issues/9035 suggests)?

Because [Suite_Builder.group](https://github.com/enso-org/enso/blob/48c17845a78f62ff244cb332d3571790b14571fb/distribution/lib/Standard/Test/0.0.0-dev/src/Suite.enso#L37) exposes `Group_Builder` to the caller, so the caller can call `Group_Builder.teardown` even if it is private.